pysparkling/sql/tests/test_casts.py

```diff
@@ -17,13 +15,14 @@
 BASE_OPTIONS = {}
 
 
-@pytest.mark.skipif(not hasattr(time, 'tzset'), reason='tzset not supported on Windows')
 class CastTests(TestCase):
     maxDiff = None
 
     def setUp(self):
         os.environ['TZ'] = 'Europe/Paris'
-        time.tzset()  # pylint: disable=no-member
+
+        if hasattr(time, 'tzset'):
```

Comment:
Do we need this `if` as we have pytest's `skipif`?

Reply:
The pytest-skipif just skipped ALL the tests on Windows. That was a bit too extensive imho.

And if I don't call the tzset(), all the tests runs fine through on Windows... Maybe because I'm also in the timezone Paris, but I did not test this.

Reply:
If all these tests just run without setting the timezone on Windows, do we need the explicit timezone on any platform?

Reply:
Unlikely. I'll try it in another branch.

Reply:
Removing the tz/set combination works just fine.
However, if I update the TZ to Asia/Shanghai, tests are failing (also in a separate process because the timezone indicator is "+08:00" instead of "+01:00").

Didn't investigate deeper on how to cover that.

And on Windows this would not work properly...
Let me think a little about this!

Reply:
Ok, thanks, not necessary to change in this PR as this was in the codebase before and you are just working around that.